scripts/install/provision.sh

```diff
@@ -106,17 +109,22 @@ deploy_dependencies() {
     SA_STATE_BACKUP_NAME="sastatebkup${prefix}${suffix}"
     SA_CONTAINER_NAME="tfstate"
 
+    #Events Symphony Resources
+    SA_EVENTS_NAME="saevents${prefix}${suffix}"
+    SA_EVENTS_TABLE_NAME="events"
+
     _information "The following resources will be Created :"
     _information ""
     _information "                     Resource Group:  $RG_NAME"
     _information "                          Key Vault:  $KV_NAME"
     _information "                 Container Registry:  $CR_NAME"
     _information "    Service Principal Name (Reader):  $SP_READER_NAME"
     _information "     Service Principal Name (Owner):  $SP_OWNER_NAME"
+    _information "            Storage account(Events):  $SA_EVENTS_NAME"
```

Comment:
can create on storage account with two different containers one for the events and another for terraform state files. the state one is only needed for symphony terraform flavor.

Reply:
Done